service/policy/db/migrations/20240805000001_drop_deprecated_resources_table.sql

```diff
@@ -0,0 +1,26 @@
+-- +goose Up
+-- +goose StatementBegin
+
+-- Remove the 'resources' table that was never used in platform 2.0 and should be removed
+
+DROP TABLE IF EXISTS resources;
+
+-- +goose StatementEnd
+
+-- +goose Down
+-- +goose StatementBegin
+
+CREATE TABLE IF NOT EXISTS resources
+(
+    id SERIAL PRIMARY KEY,
+    name VARCHAR NOT NULL,
+    namespace VARCHAR NOT NULL,
+    version INTEGER NOT NULL,
+    fqn VARCHAR,
+    labels JSONB,
+    description VARCHAR,
+    policytype VARCHAR NOT NULL,
+    resource JSONB
+);
+
```

Comment:
Could we move this to a distinct PR and update the VARCHAR to TEXT? They are the same thing but for old school DBAs it will remove the concern of specifying a length.